test/integration/fork-block-numbers.ts

```diff
@@ -1,7 +1,9 @@
 const forkBlockNumber = {
   'aave-compound-rewards': 12521999, // Ethereum
+  'asset-plugins': 14916729, // Ethereum
   'mainnet-deployment': 15690042, // Ethereum
-  default: 14916729, // Ethereum
+  'flux-finance': 16836855, // Ethereum
+  default: 16836855, // Ethereum
```

Comment:
i think @julianmrodri  said this breaks something...

Reply:
It does and I fixed it